cmd/medius/common/registry.go

```diff
@@ -27,6 +28,14 @@ var Registry = []Entry{
 		Artifact:   centos.New("8.4"),
 		UseForDocs: true,
 	},
+	{
+		Artifact:   centosstream.New("8"),
+		UseForDocs: true,
+	},
+	{
+		Artifact:   centosstream.New("9"),
```

Comment:
Maybe the order of ... centos 8, centos 7, centos-stream 9, centos streram 8 is more intuitive.